docs/articles/documentation/guides/basic-guides/assert.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: docs
+title: Assert
+permalink: /documentation/guides/basic-guides/assert.html
+---
+# Assertions
+
+You can use *assertions* to check if the tested page's state matches the expected state.
+
+TestCafe provides a comprehensive set of assertions that are based on the Behavior-Driven Development style (BDD-style).
+
+This topic consists of the following sections:
+
+* [Assertion Structure](#assertion-structure)
+* [Smart Assertion Query Mechanism](#smart-assertion-query-mechanism)
+
+## Assertion Structure
+
+To construct assertions, use the [test controller](../../reference/test-api/testcontroller/README.md)'s `expect` method.
```

Comment:
I think we should link `[expect](../../reference/test-api/testcontroller/expect/README.md)`.

And probably remove the test controller link in this case.